build/intermediate-containers.mk

```
@@ -18,15 +18,15 @@ opencv-base: opencv-base-build opencv-base-docker-per-arch
 opencv-base-build: opencv-base-build-amd64 opencv-base-build-arm32 opencv-base-build-arm64
 opencv-base-build-amd64:
 ifeq (1, ${BUILD_AMD64})
-	docker build $(CACHE_OPTION) -f $(INTERMEDIATE_DOCKERFILE_DIR)/Dockerfile.opencvsharp-build . -t $(PREFIX)/opencvsharp-build:$(BUILD_OPENCV_BASE_VERSION)-$(AMD64_SUFFIX) --build-arg PLATFORM_TAG=3.1-buster-slim
+	docker build $(CACHE_OPTION) -f $(INTERMEDIATE_DOCKERFILE_DIR)/Dockerfile.opencvsharp-build . -t $(PREFIX)/opencvsharp-build:$(BUILD_OPENCV_BASE_VERSION)-$(AMD64_SUFFIX) --build-arg PLATFORM=
```

Looks like platform is missing here

```suggestion
	docker build $(CACHE_OPTION) -f $(INTERMEDIATE_DOCKERFILE_DIR)/Dockerfile.opencvsharp-build . -t $(PREFIX)/opencvsharp-build:$(BUILD_OPENCV_BASE_VERSION)-$(AMD64_SUFFIX) --build-arg PLATFORM=-$(AMD64_SUFFIX)
```

---

I should add a comment here (and maybe remove the `build-arg` argument as well) this is intended, aspnet amd64 images are not suffixed and the other uses of the `PLATFORM` variable expect empty for amd64.